@ecomplus/storefront-components/src/js/TheProduct.js

```diff
@@ -232,7 +233,7 @@ export default {
     discount () {
       const { body } = this
       const priceValue = this.fixedPrice || getPrice(body)
-      return checkOnPromotion(body)
+      return checkOnPromotion(body) || (this.hasDiscountTagOnUtm && (body.base_price > priceValue))
```

Comment:
Isso não vai ser true só quando houver desconto por UTM, então não sei se faz sentido o nome `hasDiscountTagOnUtm`...
Fora isso, por que isso deveria ser uma prop e não só o funcionamento padrão mesmo?

Reply:
Sim, pra ser só em utm, precisaria buscar o desconto criando um evento, mas nesse caso ai falou que não precisaria fazer na outra pr que abri e sugeriu ser opcional, por isso a prop, pq teoricamente mostra a tag, se tiver base_price maior. 

No caso, fiz um teste considerando somente se this.hasDiscountTagOnUtm for true, nesse caso, quando tem por exemplo base_price = 0 ou nulo, o discount é NaN e ai lá no html, com v-if="discount > 0", não mostra de toda forma e assim não teria perigo de aparecer quando nao tiver promoção, mas o base_price for maior que zero. Teria problema o discount ficar NaN em algumas situações?

Reply:
> Sim, pra ser só em utm, precisaria buscar o desconto criando um evento

Então, mas eu não disse que teria que ser só pra UTM na verdade, o que eu quis dizer é que o nome da prop tá ruim porque o que ela faz não tá diretamente relacionado a UTM...

Reply:
É tipo um restaurante perguntar se o cliente quer salada, e se ele responder que sim o restaurante pode adicionar a salada ou um vinagrete, caso não tenha a salada kkk
Você está alterando a API do componente, e aí nomenclatura é importante

Reply:
Beleza, nome está errado mesmo, vou alterar